docs/doxygen/migration/jobs.dox

```diff
@@ -0,0 +1,35 @@
+/**
+@page jobs_migration AWS IoT Jobs
+@brief How to migrate an application using AWS IoT Jobs from v3 to 202009.00.
```

Comment:
```suggestion
@brief How to migrate an application using AWS IoT Jobs from v3 to 202009.00 and subsequent releases.
```

Reply:
We cannot use **202009.00** as it did not contain the Jobs library.

Reply:
```suggestion
@brief How to migrate an application using AWS IoT Jobs from v3 to 202011.00 and subsequent releases.
```